pkg/node/helper.go

```diff
@@ -302,16 +302,12 @@ func GetClusterNameFromList(app *application.Avalanche) (string, error) {
 // GetAvalancheGoVersion asks users whether they want to install the newest Avalanche Go version
 // or if they want to use the newest Avalanche Go Version that is still compatible with Subnet EVM
 // version of their choice
-func GetAvalancheGoVersion(app *application.Avalanche, avagoVersion AvalancheGoVersionSettings) (string, error) {
+func GetAvalancheGoVersion(app *application.Avalanche, avagoVersion AvalancheGoVersionSettings, network models.Network) (string, error) {
 	// skip this logic if custom-avalanchego-version flag is set
```

Comment:
the function to prompt avalanche should be either moved to pkg/prompts or prompt.go on pkg/dependencies, or reuse a new function used for all dependencies, under pkg/prompts.
the options may be, as already said:
- supported
- latest
- prerelease
- version

Reply:
see above